tests/autoclassify/utils.py

```diff
@@ -66,35 +71,30 @@ def get_data(base_data, updates):
     return data
 
 
-def create_summary_lines_failures(project, job, failure_line_list):
-    formatter = TbplFormatter()
-
-    text_log_summary = {"logurl": "http://example.org/log-test/",
-                        "step_data": {"all_errors": [],
-                                      "steps": [],
-                                      "errors_truncated": False},
-                        }
+def create_text_log_errors(project, job_id, failure_line_list):
+    job = Job.objects.get(repository__name=project,
+                          project_specific_id=job_id)
+    step = TextLogStep.objects.create(
+        job=job,
+        name='everything',
+        started_line_number=1,
+        finished_line_number=10,
+        started=datetime.datetime.now(),
+        finished=datetime.datetime.now(),
+        result=TextLogStep.TEST_FAILED)
 
+    formatter = TbplFormatter()
+    errors = []
     for i, (base_data, updates) in enumerate(failure_line_list):
         data = get_data(base_data, updates)
         if not data:
             continue
+        error = TextLogError.objects.create(job=job, step=step,
+                                            line=formatter(data).split("\n")[0],
```

Comment:
you could also use ``rstrip()`` to remove the newlines or any trailing whitespace.